Library/Homebrew/startup/bootsnap.rb

```diff
@@ -1,11 +1,13 @@
 # typed: false
 # frozen_string_literal: true
 
+# Disable Rails cops, as we haven't required active_support yet.
+# rubocop:disable Rails
```

Comment:
This is tangential, but I've disabled Rails cops in this file based on the comments on the previous disables below